estate/models/estate_property.py

```diff
@@ -0,0 +1,117 @@
+from odoo import api, models, fields, exceptions, tools
```

Comment:
```suggestion
from odoo import api, fields, models
from odoo.exceptions import UserError, ValidationError
from odoo.tools import float_compare, float_is_zero
```
We usually directly import these functions from their library such that we do not have to call the library every time